Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingBehaviour.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using BForBoss.Utility;
+using PerigonGames;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class RingBehaviour : MonoBehaviour
+    {
+        public event Action<RingBehaviour> RingActivated;
+        
+        [SerializeField] private TMPro.TMP_Text[] _labels;
+        [SerializeField] private string _label;
```

Comment:
If we have a tmp text, i don't think we need this.

Reply:
This was a convenience thing, just used by the on validate. Since its annoying to have to set the same text in multiple places

Reply:
What's the text you have to set?

Reply:
The number on the ring, its two different tmp objects with the same label

Reply:
Okay i see, 
Then you can probably just `FindChildComponent` for the TMP_Text.

Reply:
Good call yeah